azimuth_capi/models/v1alpha1/cluster.py

```diff
@@ -156,6 +156,10 @@ class ClusterSpec(schema.BaseModel):
         default_factory = AddonsSpec,
         description = "Describes the optional addons that should be enabled for the cluster."
     )
+    createdByUsername: schema.Optional[str] = None
```

Comment:
We probably need the description here? Do I just add the Field property in here?

Reply:
Using a `Field` allows you to specify a description, yes.